src/box/lua/tuple_format.lua

```diff
@@ -0,0 +1,11 @@
+-- new() needs a wrapper in Lua, because format normalization needs to be done
+-- in Lua.
+box.tuple.format.new = function(fmt)
+    if fmt == nil then
+        return box.internal.tuple_format.new()
+    else
+        box.internal.check_param(fmt, 'format', 'table')
+        fmt = box.internal.space.normalize_format(nil, "unknown", fmt)
```

Comment:
There may actually be a space named 'unknown' so this looks suspicious.

Reply:
Fixed, move this to foreign key name creation as we discussed.